File: Examples/SimpleGet.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using WinHTTPSharp;
using WinHTTPSharp.WinHttp;

namespace Examples
{
    public class SimpleGet
    {
        public static void _SimpleGet()
        {
            IWinHttpRequest whttp = new WinHttpRequest();

            whttp.Open("GET", "https://www.google.co.kr");
            whttp.Send();

            Console.WriteLine(whttp.ResponseText);
        }

        public static async Task _SimpleGetAsync()
        {
            IWinHttpRequest whttp = new WinHttpRequest();
            whttp.Open("GET", "https://www.google.co.kr", true);
            await whttp.Send();

            Console.WriteLine(whttp.ResponseText);
        }
    }
}

File: Examples_VB/SimpleGet.vb
Imports WinHTTPSharp.WinHttp
Module SimpleGet
    Public Sub _simpleGet()
        Dim wHttp As WinHttpRequest = New WinHttpRequest

        wHttp.Open("GET", "https://www.google.co.kr")
        wHttp.Send()

        Console.WriteLine(wHttp.ResponseText)
    End Sub
End Module
